charmhelpers/contrib/openstack/context.py

```diff
@@ -1598,6 +1598,10 @@ def __call__(self):
                 'rel_key': 'enable-nsg-logging',
                 'default': False,
             },
+            'global-physnet-mtu': {
```

Comment:
I suggest that we also pass physical-network-mtus because MTU can be overridden on a per-physnet basis.

Reply:
Makes sense, thanks.